Azure/communication-ui-library-android
# [Features] hang up on call

## Purpose

* Hang up on call accepted

## Does this introduce a breaking change?

```
[] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test


* Accept the "Read phone state" permission (should show on setup screen)
* Join a Teams/Groups meeting
* Receive & Accept a PTSN (I.e. regular phone call)
* Observe hangup of Teams/Groups meeting

## What to Check
Verify that the following are valid
- When phone_state granted, calls hang up on incoming accepted call
- When phone_state denied, hang up feature gracefully degrades (app is stable, but no hang up).

## Other Information
